tests/testAllObjects.py

```diff
@@ -47,6 +48,8 @@ class TestCat(InstanceCatalog):
 
 class basicAccessTest(unittest.TestCase):
 
+    longMessage = True
```

Comment:
longMessage is a unittest parameter that, if set to True, means that, if you call ```self.assertEqual(x, y, msg='this is my message')```, then `this is my message' will be appended to the default error message when the assert fails (as opposed to completely replacing the default error message if ```longMessage=False```).